Userland/Libraries/LibWeb/CSS/Parser/DeprecatedCSSParser.cpp

```diff
@@ -279,7 +280,7 @@ static StringView isolate_calc_expression(const StringView& value)
     return value.substring_view(5, substring_length);
 }
 
-static CSS::CalculatedStyleValue::CalcSum parse_calc_expression(const StringView& expression_string)
+static Optional<NonnullOwnPtr<CSS::CalculatedStyleValue::CalcSum>> parse_calc_expression(const StringView& expression_string)
```

Comment:
Use `OwnPtr<T>` instead of `Optional<NonnullOwnPtr<T>>`, there's no reason to have extra overhead here (unless you like the expressiveness, which is understandable)

Reply:
This doesn't seem to have been addressed?